stats.py

```diff
@@ -61,10 +61,10 @@ def standard_deviation(number_list):
         assert isinstance(num, numbers.Number), "We need numbers yo."
     assert len(number_list) > 1, "Please enter at least 2 numbers to calculate standard deviation."
     #function
-    answer = statistics.stdev(number_list)
+    mean_ = mean(number_list)
+    variance = sum([(e-mean_) ** 2 for e in number_list]) / len(number_list)
+    answer = math.sqrt(variance)
     rounded_answer = round(answer, 5)
     #postconditions
-    assert isinstance(rounded_answer, numbers.Number), "Standard Deviation will return a number."
-    assert rounded_answer > min(number_list), "SD should be higher than lowest number in list."
```

Comment:
Although this is not the correct sanity check to be making, there is still a reasonable sanity check that could be made here.